### PR TITLE
Fix the issue for poll chat

### PR DESCRIPTION
**Pull request by timur-ul-hasan**
Fix the issue for the poll chat
Click up Issue:
https://app.clickup.com/t/86b0tcexu

### CODE REVIEW

File: flowback/chat/views.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def get(self, request, channel_id: int):
         serializer = self.FilterSerializer(data=request.query_params)
         serializer.is_valid(raise_exception=True)
 
-        messages = message_list(user=request.user, channel_id=channel_id, filters=serializer.validated_data)
+        messages = message_list(user=request.user, channel_id=channel_id, **serializer.validated_data)
```
Comment (Gofven) [inline flowback/chat/views.py:35]: While this is good, it's better to set filters=None to message_list function aswell as the other selectors present in the file c:

File: flowback/poll/services/poll.py [hunk 2]
```diff
@@ -137,6 +140,14 @@ def poll_create(*, user_id: int,
                                   category='poll_schedule',
                                   message=f'Poll {poll.title} has finished, group schedule has been updated',
                                   related_id=poll.id)
+    
+    topic = message_channel_topic_create(
+        channel_id=group_user.group.chat_id,
+        topic_name=f'Poll.{poll.id}',
+    )
+
+    poll.message_channel_topic = topic
```
Comment (Gofven) [inline flowback/poll/services/poll.py]: Conflicts with Poll pre_save signal, if you wish to change this, check flowback/poll/models.py line 240 c:

**Review by Gofven — CHANGES_REQUESTED** [review]
It's mostly a good commit, although there are some interfering issues that you might need to check before i accept this pull request c:

File: flowback/poll/services/poll.py (hunk 2 above)
Comment (the-hasan-javed) [reply flowback/poll/services/poll.py]: Thanks @Gofven 
Is the signal working properly, then we can use that signal.

Comment (Gofven) [reply flowback/poll/services/poll.py]: Yes it does work properly!

**Comment by timur-ul-hasan** [comment]
@Gofven 
Please review again conflict resolved.

**Review by Gofven — APPROVED** [review]

**Pull request merged**